src/main/java/org/folio/bulkops/service/BulkOperationService.java

```diff
@@ -301,26 +312,35 @@ public void confirm(BulkOperation operation)  {
 
       operation.setLinkToModifiedRecordsJsonFile(modifiedJsonFileName);
 
-      dataProcessing.setProcessedNumOfRecords(processedNumOfRecords);
-      dataProcessingRepository.save(dataProcessing);
+      updateProcessed(dataProcessing, operation, processedNumOfRecords);
 
-      operation.setApproach(IN_APP);
-      operation.setStatus(OperationStatusType.REVIEW_CHANGES);
-      operation.setProcessedNumOfRecords(processedNumOfRecords);
       bulkOperationRepository.findById(operation.getId()).ifPresent(op -> operation.setCommittedNumOfErrors(op.getCommittedNumOfErrors()));
+    } catch (S3ClientException e) {
+      errorMsg = ERROR_NOT_CONFIRM_CHANGES_S3_ISSUE;
```

Comment:
The same as https://github.com/folio-org/mod-bulk-operations/pull/316/files#r1855540575